### ISSUE TITLE
Odoo13 - Ubuntu 18.04.1 - Installation problem

### ISSUE THREAD

**Issue by byk3v** [Yenthe666/InstallScript#231]
User : root
Folder : /odoo/

Hi there,

During the execution of the script I found these error messages :

       Permission denied

`---- Create custom module directory ----

mkdir: cannot create directory ‘/odoo/custom’: Permission denied
mkdir: cannot create directory ‘/odoo/custom/addons’: No such file or directory

Any SOlution_?

Thanks in advance

**Comment by chris001** [comment]
1. What is the type of server you're attempting to install on?  real linux server, virtual kvm server, virtual openvz server, virtual lxc server, google cloud server, amazon aws server, virtualbox, etc ?
2. What is the output of the command: `whoami` ?

**Comment by byk3v** [comment]
There was a permission Issue, but after uninstalling packages, I faced troubles with postgresql, and decided to even reinstall Ubuntu. With my fresh OS, I ran the script, and still got the 'role "odoo" does not exist ERROR', but after:
1) sudo su postgres
2) createuser odoo -s
3) psql template1
4) \q
5) exit

My problem seemed solved but I got still a: "psycopg2.programmingerror permission denied to create database"

I wrote:

1. sudo su  postgres    (the idea is to switch to postgresql user)

2. psql

3. ALTER USER odoo WITH CREATEDB;

And it works Fine now!!... Thanks a lot
